cognite/transformations_cli/commands/deploy/transformations_api.py

```diff
@@ -224,13 +225,13 @@ def upsert_transformations(
 
         for u in chunk_items(items_to_update):
             client.transformations.update(u)
-            # Partial update for data set id to be able to clear data set id field when requested.
-            dataset_update = [
-                # Clear data set id if it is not provided, else set data set id with a new value
-                TransformationUpdate(external_id=du.external_id).data_set_id.set(du.data_set_id)
+            # Partial update for data set id and tags to be able to clear data set id field when requested.
+            partial_update = [
+                # Clear data set id and tags if they are not provided, else set data set id and tags with a new value
+                TransformationUpdate(external_id=du.external_id).data_set_id.set(du.data_set_id).tags.set(du.tags)
```

Comment:
not a big deal, but why are we making multiple API calls here? couldn't we just assign
```
u.data_set_id = du.data_set_id
u.tags = du.tags
```
and then do a single 
 ```
 client.transformations.update(items_to_update)
 ```

Reply:
I think it is only one, it adds setter to the same body. this allows setNull

Reply:
It was adding to the same body when i printed at least :P

Reply:
this change may not be needed actually, i'll check

Reply:
hehe.... it's two calls per transformation, one in line 227, and another on line 234 😅 

not a big deal for now.... but it could start having a performance impact on repos with 100s of transformations.